downstream/modules/platform/proc-deploy-eda-controller-with-aap-operator-ocp.adoc

```diff
@@ -16,7 +16,26 @@
 
 . Under *Provided APIs*, locate the {EDAName} modal and click *Create instance*. 
 +
-This takes you to the Form View to customize your installation.
+This takes you to the Form View to customize your installation. 
++
+[IMPORTANT]
+====
+To ensure that you can run concurrent {EDAName} activations efficiently, you must set your maximum number of activations in proportion to the resources available on your cluster. You can do this by adjusting your {EDAName} settings in the YAML view. 
+
+When activating a rulebook under standard conditions, it typically utilizes approximately 250 MB of memory. However, the actual memory consumption can vary significantly based on the complexity of your rules and the volume and size of the events processed. In scenarios where a large number of events are anticipated or the rulebook complexity is high, conduct a preliminary assessment of resource usage in a staging environment. This ensures that your maximum number of activations is based on the capacity of your resources. 
+====
++
```

Comment:
I have added the paragraph suggested by @Alex-Izquierdo with a few tweaks that are necessary due to IBM and RH style guide stipulations. @Alex-Izquierdo and @rooftopcellist - please review the last statement beginning with "This ensures that your maximum number..." I had to remove the "we recommend" from the statement, per style guide rules. (Notes state the following: "In most cases, avoid "recommending" that users do something, especially if the recommendation is vague or unexplained. Instead, describe why the action is problematic or beneficial."